python/pip-wheel-metadata/pyarrow.dist-info/LICENSE.txt

```diff
@@ -0,0 +1,88 @@
+## 3rd-party licenses for code that has been adapted for the Arrow Python
```

Comment:
is this change required for this PR?  If not maybe add it as a separate JIRA/PR

Reply:
No.  It's removed.